vms/vms/urls.py

```diff
@@ -16,4 +22,7 @@
     url(r'^registration/', include('registration.urls', namespace='registration')),
     url(r'^shift/', include('shift.urls', namespace='shift')),
     url(r'^volunteer/', include('volunteer.urls', namespace="volunteer")),
+    url(r'^portal', TemplateView.as_view(template_name='home/home.html'), name='home'),
+    url(r'^login/$', anonymous_required(auth_views.login), {'template_name': 'authentication/login.html'}, name='login_process'),
```

Comment:
I frequently come across [this](https://travis-ci.org/systers/vms/builds/137914904#L5806) exception in the build. If the url changes are not necessary, please revert them back to the initial ones. Also, I think there is a url already mapping to the home page in the urls.py. Isn't the `portal/` url linking to the same template?